src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -1024,6 +1043,23 @@ namespace GraphQL.Execution
         protected virtual bool ShouldIncludeNode(GraphQL.Execution.ExecutionContext context, GraphQLParser.AST.IHasDirectivesNode node) { }
         protected virtual void ValidateNodeResult(GraphQL.Execution.ExecutionContext context, GraphQL.Execution.ExecutionNode node) { }
     }
+    public class ExecutionStrategyRegistration : System.IEquatable<GraphQL.Execution.ExecutionStrategyRegistration>
```

Comment:
> > Let's review and discuss again what should be removed/obsolete/remain.
> 
> I'm fine with current state. I can say that I like it more to delete the old code than to mark it as obsolete.

Just so you understand my thought process, using SubscriptionDocumentExecuter (a class we provide) can break AddExecutionStrategy (a method we provide). So that’s why I marked it as obsolete. Whereas the SelectExecutionStrategy does not interfere unless overridden, in which case the user presumably wanted to override the behavior.